backend/app/assets/javascripts/spree/backend/global/tinymce.es6

```diff
@@ -2,10 +2,11 @@ document.addEventListener('DOMContentLoaded', function() {
   // eslint-disable-next-line no-undef
   tinymce.init({
     selector: '.spree-rte',
+    images_upload_url: '/admin/uploader/image',
```

Comment:
@MatthewKennedy let's use Platform API for this, please :) 

`POST /api/v2/platform/images`